jdi-light-html-tests/src/main/java/io/github/com/pages/Header.java

```diff
@@ -4,33 +4,44 @@
 import com.epam.jdi.light.elements.composite.Form;
 import com.epam.jdi.light.elements.composite.Section;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
-import com.epam.jdi.light.ui.html.elements.common.*;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Icon;
+import com.epam.jdi.light.ui.html.elements.common.Image;
 import io.github.com.custom.LoginFormSmart;
 import io.github.com.entities.User;
 import io.github.com.sections.LoginForm;
 import org.openqa.selenium.WebElement;
 
 public class Header extends Section {
-	// @UI("form")
-	public static LoginForm loginForm;
-	public static LoginFormSmart loginFormSmart;
-	@UI("#login-form") public static Form<User> loginFormLight;
-	@UI("#login-form") public static Form<User> loginFormLightLocator;
-	@UI(".fa-sign-out") public static Button logout;
-	public static Icon userIcon;
-	public static Icon userName;
-	public static Image epamLogo;
 
-	@UI(".search-field input") static UIElement searchInput;
-	@UI(".icon-search") static WebElement searchIcon;
-	@UI(".icon-search.active") static WebElement doSearch;
+    // @UI("form")
+    public static LoginForm loginForm;
+    public static LoginFormSmart loginFormSmart;
+    @UI("#login-form")
+    public static Form<User> loginFormLight;
+    @UI("#login-form")
+    public static Form<User> loginFormLightLocator;
+    @UI(".fa-sign-out")
+    public static Button logout;
+    public static Icon userIcon;
+    public static Icon userName;
+    public static Image epamLogo;
 
-	public static void search(String text) {
-		if(logout.isDisplayed())
-			userName.click();
-		if (searchInput.isHidden())
-			searchIcon.click();
-		searchInput.input(text);
-		doSearch.click();
-	}
-}
\ No newline at end of file
+    @UI(".search-field input")
+    static UIElement searchInput;
+    @UI(".icon-search")
+    static WebElement searchIcon;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5540607)